examples/k8s/deployment.yaml

```diff
@@ -113,10 +116,10 @@ webhooks:
       port: 8443
       namespace: "devops"
   sideEffects: None
-  admissionReviewVersions: ["v1beta1"]
+  admissionReviewVersions: ["v1", "v1beta1"]
```

Comment:
Adding `v1` [AdmissionReviewVersion](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/#webhook-request-and-response) for completeness